src/GazeboUnderwater.cpp

```diff
@@ -115,50 +119,78 @@ namespace gazebo_underwater
         buoyancy = abs(buoyancy) + computeModelMass(model) * world->GetPhysicsEngine()->GetGravity().GetLength();
         centerOfBuoyancy = getParameter<math::Vector3>("center_of_buoyancy","meters",
                 math::Vector3(0, 0, 0.15));
+        std::string damp_matrices;
+        damp_matrices += "50 0 0 0 0 0\n";
+        damp_matrices += "0 50 0 0 0 0\n";
+        damp_matrices += "0 0 50 0 0 0\n";
+        damp_matrices += "0 0 0 45 0 0\n";
+        damp_matrices += "0 0 0 0 45 0\n";
+        damp_matrices += "0 0 0 0 0 45\n\n";
+        damp_matrices += "40 0 0 0 0 0\n";
+        damp_matrices += "0 40 0 0 0 0\n";
+        damp_matrices += "0 0 40 0 0 0\n";
+        damp_matrices += "0 0 0 35 0 0\n";
+        damp_matrices += "0 0 0 0 35 0\n";
+        damp_matrices += "0 0 0 0 0 35";
+        dampingCoefficients = convertToMatrices(getParameter<string>("damping_coefficients","N/vel^n / vel^n={m/s, rad/s, m2/s2. rad2/s2}",damp_matrices));
+
+        std::string extra_inertia;
+        extra_inertia += "0 0 0 0 0 0\n";
+        extra_inertia += "0 0 0 0 0 0\n";
+        extra_inertia += "0 0 0 0 0 0\n";
+        extra_inertia += "0 0 0 0 0 0\n";
+        extra_inertia += "0 0 0 0 0 0\n";
+        extra_inertia += "0 0 0 0 0 0";
+        addedInertia = convertToMatrix(getParameter<string>("added_inertia","Kg, Kg.m2", extra_inertia));
+
+        gzInertia = mountGzInertiaMatrix(model);
+        inverseInertia = gzInertia + addedInertia;
```

Comment:
Avoid assigning something to a variable that is *not* what it's meant to hold _in fine_. Just create a temporary completeInertia and inverse it in inverseInertia